src/styles/index.scss

```diff
@@ -66,3 +66,4 @@ $btn-padding-x:               2.8rem;
 @import "../sections/Join/style";
 @import "../sections/Notification/style";
 @import "../sections/FAQ/style";
+@import "../sections/CTA/style";
```

Comment:
@duranmla I think we can import these in the component itself, instead of having just one file with CSS. In this case is not different because we only have 1 page, but for multiple pages/components is a good idea to do it.

Reply:
Probably the comment above is not explicit enough but we need to add this imports here in order to be able to use all the bootstrap variables as expected.